tests/Zend/Service/Amazon/OnlineTest.php

```diff
@@ -327,26 +334,3 @@ public function testQueryAsin()
         $this->assertTrue($item instanceof Zend_Service_Amazon_Item);
     }
 }
-
-
-/**
- * @category   Zend
- * @package    Zend_Service_Amazon
- * @subpackage UnitTests
- * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
- * @license    http://framework.zend.com/license/new-bsd     New BSD License
- * @group      Zend_Service
- * @group      Zend_Service_Amazon
- */
-class Zend_Service_Amazon_OnlineTest_Skip extends PHPUnit_Framework_TestCase
```

Comment:
These aren't used now, and if left in the file, will be executed by PHPUnit when the file is loaded (not harmful, just wasted time/message/resources).